tap_github/repository_streams.py

```diff
@@ -1488,6 +1488,13 @@ class StargazersStream(GitHubRestStream):
     # GitHub is missing the "since" parameter on this endpoint.
     missing_since_parameter = True
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # TODO - remove warning with next release.
+        self.logger.warning(
+            "This stream is deprecated. Please use the Graphql version instead 'stargazers'."
+        )
```

Comment:
@ericboucher this will be shown even if folks are not selecting (i.e. in the catalog) this stream, is that the okt?

Reply:
hmm good catch where would you put it?